library/kotlin/io/envoyproxy/envoymobile/PulseClient.kt

```diff
@@ -13,12 +13,32 @@ interface PulseClient {
    */
   fun counter(vararg elements: Element): Counter
 
+  /**
+   * @return A counter based on the joined elements with list of tags.
+   */
+  fun counter(vararg elements: Element, tags: List<Tag>): Counter
```

Comment:
Do Kotlin interfaces support optional params? If so we could simplify and just have one method like
```kotlin
fun counter(vararg elements: Element, tags: List<Tag> = emptyList()): Counter
```

you might need to add `JvmOverloads` for java interop